lib/ansiblereview/examples/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
This file might be more appropriate where it's actually used vs. here.

Reply:
Probably `config.ini` as well.

Reply:
Removed.